### PR TITLE
IBD work within/between regions

### PR DESCRIPTION
**Pull request by hardingnj**
No differential IBD between/within sites in Burkina Faso for gambiae and coluzzii.

### CODE REVIEW

**Comment by cclarkson** [comment]
Hi @hardingnj, is this waiting for a review?

**Comment by hardingnj** [comment]
Not just yet- I think I'll run another population through it first. Very happy to hear comments though!

**Comment by hardingnj** [comment]
Hi @cclarkson ready for review now. 
Nothing looks notable- so I don't think it's a strong candidate for inclusion as is. Happy to discuss at the next meeting though! 
If we are not including it- no need to merge I guess? Or maybe we can merge then remove it, that way we at least have a record.

**Comment by hardingnj** [comment]
This is complete. 
It turned out that I made a rick in my initial analysis of Kenya. By not sorting the sample pairs, where I had the wrong order, I set the IBD sharing to zero. This biased my results in favour of the cross sample group, as they were always sorted correctly within groups. 

TLDR: No differential effect of IBD across regions, that I can detect here at least.

**Comment by cclarkson** [comment]
Hi, @hardingnj, does the Kenya IBD result now recapitulate that you found in phase ?

**Comment by hardingnj** [comment]
No- but I know why now. I made a error in the initial phase 1 analysis- this is now corrected. 

(Although it doesn't confirm either way whether we can trust the p2 kenya locations.)
